tags/help-wanted.md

```diff
@@ -0,0 +1,6 @@
+---
+name: help-wanted
+displayName: Help Wanted
+description: Help wanted for the issue
```

Comment:
The tags here are referring to tags on repos, not labels on issues

for example on vscode repo the tags are

'editor', 'electron', etc

![image](https://user-images.githubusercontent.com/5168796/47751026-f709b080-dc4d-11e8-9048-b438af034320.png)

So help-wanted tag on a repo would not be describing a particular issue, it would be describing the whole project.

Does that make sense? Would you like to update the description for this?